package.json

```diff
@@ -25,5 +28,15 @@
     "mkdirp": "^0.5.1",
     "nodeos-mount": "^0.2.0",
     "posix": "^4.0.0"
+  },
+  "devDependencies": {
+    "chai": "^3.5.0",
+    "errno-codes": "^1.0.2",
+    "istanbul": "^0.4.3",
+    "mocha": "^2.4.5",
+    "proxyquire": "^1.7.4",
+    "rimraf": "^2.5.2",
+    "sinon": "^1.17.3",
+    "sinon-chai": "^2.8.0"
```

Comment:
@piranna Not all of them but chai, errno-codes, istanbul, mocha, proxyquire, sinon and sinon-chai are required for testing